### PR TITLE
Support for Boaty

### PR DESCRIPTION
**Pull request by Jels02**
## Description

Support for the Boaty - the boat shaped keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jels/boaty/config.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x006a     //jels
```
Comment (drashna) [inline keyboards/jels/boaty/config.h:22]: While this vendor ID doesn't appear to be in use, I am very uncomfortable about using such a low ID. 

Ideally, anything past 0x4000 should be okay, and the further past, the better.

File: keyboards/jels/boaty/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Boaty
+
+![Boaty](https://cdn.shopify.com/s/files/1/0582/4420/2650/products/boaty_compressed_2048x2048.jpg)
```
Comment (drashna) [inline keyboards/jels/boaty/readme.md:3]: Please rehost this image on Imgur or a similar dedicated image hosting site.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/jels/boaty/config.h (hunk 1 above)
Comment (Jels02) [reply keyboards/jels/boaty/config.h:22]: I'd be happy to change it, but I'd also have to update my other boards (easy enough) and then also update via (could take a while) and they would be unrecognized in the down time

File: keyboards/jels/boaty/config.h [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x6a6b     //jels
+#define PRODUCT_ID      0x000b     //Boaty
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    jels
+#define PRODUCT         boaty
```
Comment (drashna) [inline keyboards/jels/boaty/config.h:26]: This should be moved to the info.json file.

File: keyboards/jels/boaty/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Boaty
+
+![Boaty](https://i.imgur.com/6LEaYxS.jpg)
```
Comment (drashna) [inline keyboards/jels/boaty/readme.md:3]:
```suggestion
![Boaty](https://i.imgur.com/6LEaYxSh.jpg)
```

File: keyboards/jels/boaty/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+*   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+//base layer
+[0] = LAYOUT_all(
+                                            KC_NLCK,
```
Comment (fauxpark) [inline keyboards/jels/boaty/keymaps/via/keymap.c:23]:
```suggestion
                                            KC_NUM,
```

File: keyboards/jels/boaty/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+*   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+//base layer
+[0] = LAYOUT_all(
+                                            KC_NLCK,
```
Comment (fauxpark) [inline keyboards/jels/boaty/keymaps/default/keymap.c:23]:
```suggestion
                                            KC_NUM,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**